pytest/tests/loadtest/locust/download_contracts.sh

```diff
@@ -2,8 +2,7 @@
 #
 # Downloads the WASM contracts necessary for all workloads and stores them in "res" folder.
 
```

Comment:
You may want to consider cleaning up the previous versions of those files before re-downloading / re-copying them.